src/extra/normalize.css

```diff
@@ -37,6 +37,21 @@
   outline-offset: 5px;
 }
 
+:where(a, button, input, textarea, summary) {
+  --outline-size: max(5px, 1px);
+  --outline-style: solid;
+  --outline-color: var(--blue-5);
+}
+
+:where(a, button, input, textarea, summary):focus-visible {
+  outline: var(--outline-size) var(--outline-style) var(--outline-color);
+  outline-offset: var(--outline-offset, var(--outline-size));
```

Comment:
where is `--outline-offset` defined?

Reply:
if --outline-offset missed or not define by user the second param will used instead

Reply:
In case redefine offset we can do like this
```css
input:focus-visible {
  --outilne-offset: 5px;
}
```